### PR TITLE
Remove most remaining usages of IFluidHTMLView

### PR DESCRIPTION
**Pull request by ChumpChief**
This removes usage of IFluidHTMLView from Codemirror, Prosemirror, and Shared-text.  It does so by locally wrapping the existing view in a React element (thereby hiding the render() contract from consumers of the react view).

Smde remains, which is currently broken in main.  Once that is fixed I can remove the usage there too using the same strategy.

### CODE REVIEW

File: examples/data-objects/codemirror/src/index.ts [hunk 1]
```diff
@@ -15,11 +15,14 @@ import {
 	RuntimeFactoryHelper,
 } from "@fluidframework/runtime-utils";
 import { MountableView } from "@fluidframework/view-adapters";
+
+import React from "react";
+
```
Comment (tylerbutler) [inline examples/data-objects/codemirror/src/index.ts:20]: nit: is the extra spacing here intentional? Leave it to me to find the one thing prettier doesn't deal with... 😩

File: examples/data-objects/prosemirror/src/index.ts [hunk 2]
```diff
@@ -15,9 +15,12 @@ import {
...
 } from "@fluidframework/runtime-utils";
 import { MountableView } from "@fluidframework/view-adapters";
-import { ProseMirror, ProseMirrorFactory, ProseMirrorView } from "./prosemirror";
 
-export { ProseMirror, ProseMirrorFactory, ProseMirrorView } from "./prosemirror";
+import React from "react";
+
+import { ProseMirror, ProseMirrorFactory, ProseMirrorReactView } from "./prosemirror";
+
+export { ProseMirror, ProseMirrorFactory, ProseMirrorReactView } from "./prosemirror";
```
Comment (tylerbutler) [inline examples/data-objects/prosemirror/src/index.ts:23]: Spacing, plus looks like dupe imports?

File: examples/data-objects/codemirror/src/index.ts (hunk 1 above)
Comment (ChumpChief) [reply examples/data-objects/codemirror/src/index.ts:20]: Prettier doesn't seem to care either way - typically I prefer to use newlines to split into three groups of imports:

```ts
import { thing1 } from "@fluidframework/package1";
import { thing2 } from "@fluidframework/package2";

import { thing3 } from "non-fluid-package1";
import { thing4 } from "non-fluid-package2";

import { thing5 } from "./localReference1";
import { thing6 } from "./localReference2";
```

Not a strong feeling though if you'd prefer the first two groups get merged.

File: examples/data-objects/prosemirror/src/index.ts (hunk 2 above)
Comment (ChumpChief) [reply examples/data-objects/prosemirror/src/index.ts:23]: Prettier doesn't care - I can remove the newline if you like.

Some of the older demos like this one do a weird import/export thing which is what's happening here.

Comment (tylerbutler) [reply examples/data-objects/prosemirror/src/index.ts]: I completely missed that one was export. Carry on.

File: examples/data-objects/codemirror/src/index.ts (hunk 1 above)
Comment (tylerbutler) [reply examples/data-objects/codemirror/src/index.ts:20]: Yeah, prettier only condenses multiple line breaks into one, nothing more. I can't wait for the code that looks like this:

```ts
if (condition === true) {

    const foo = bar;

    console.log(`foo: ${bar}`);

}
```

😝

**Review by tylerbutler — APPROVED** [review]
:shipit:

**Pull request merged**